test/conformance/runtime/readiness_probe_test.go

```diff
@@ -30,38 +30,67 @@ import (
 	v1a1test "knative.dev/serving/test/v1alpha1"
 )
 
+// testCases for table-driven testing.
+var testCases = []struct {
```

Comment:
nit: Do you plan to reuse these in a different test? If not, i'd suggest to inline into the using test to declutter the file.

Reply:
That makes sense. Moved the testcases into `TestProbeRuntime()`.